### PR TITLE
Add CPU ram usage

### PR DESCRIPTION
**Pull request by Rizhiy**
Add CPU RAM usage using `free` command.

### CODE REVIEW

**Comment by Rizhiy** [comment]
Sorry if my string manipulation is a bit junky, not very experienced with that stuff.

**Comment by casperdcl** [comment]
> Sorry if my string manipulation is a bit junky, not very experienced with that stuff.

No problem; cleaned it up.

File: scripts/ram_percentage.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+source "$CURRENT_DIR/helpers.sh"
+
+ram_percentage_format="%3.1f%%"
+
+print_ram_percentage() {
+  ram_percentage_format=$(get_tmux_option "@ram_percentage_format" "$ram_percentage_format")
+
+  if command_exists "free"; then
+    free | awk -v format="$ram_percentage_format" '$1 == "Mem:" {printf(format, 100*$3/$2)}'
+  fi
```
Comment (casperdcl) [inline scripts/ram_percentage.sh]: @ctjhoa doyou think we should have an `else echo "free not found"` message? I think not. Would be better to have an `elif command_exists <some_other_cmd>`...

Comment (Rizhiy) [reply scripts/ram_percentage.sh]: You can also get memory information from `/proc/meminfo` as a backup, although I'm not sure whether it is cross-distribution.

Comment (casperdcl) [reply scripts/ram_percentage.sh]: > **free**  displays the total amount of free and used physical and swap memory in the system, as well as the buffers and caches used by  the  kernel.  The  information  is  gathered by parsing `/proc/meminfo`.

So it looks like `free` should be supported anywhere `/proc/meminfo` is available.

Comment (Rizhiy) [reply scripts/ram_percentage.sh]: Another alternative is `vmstat`, but I'm pretty sure `free` should be available on vast majority Linux distros.

File: scripts/ram_bg_color.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+get_bg_color_settings() {
+  ram_low_bg_color=$(get_tmux_option "@ram_low_bg_color" "$ram_low_default_bg_color")
+  ram_medium_bg_color=$(get_tmux_option "@ram_medium_bg_color" "$ram_medium_default_bg_color")
+  ram_high_bg_color=$(get_tmux_option "@ram_high_bg_color" "$ram_high_default_bg_color")
+}
+
+print_bg_color() {
+  local ram_percentage=$($CURRENT_DIR/ram_percentage.sh | sed -e 's/%//')
+  local ram_load_status=$(cpu_load_status $ram_percentage)
```
Comment (ctjhoa) [inline scripts/ram_bg_color.sh]: `cpu_load_status` should be renamed at some point

File: scripts/ram_percentage.sh (hunk 1 above)
Comment (ctjhoa) [reply scripts/ram_percentage.sh]: I'm ok without `else` but we should precise `free` requirement in the README.

File: scripts/ram_bg_color.sh (hunk 2 above)
Comment (ctjhoa) [reply scripts/ram_bg_color.sh]: Not necessarily in this PR

Comment (casperdcl) [reply scripts/ram_bg_color.sh]: Yes probably best to fix later.

File: scripts/ram_percentage.sh (hunk 1 above)
Comment (casperdcl) [reply scripts/ram_percentage.sh]: done now

**Pull request merged**